packages/schematics/src/convert-tslint-to-eslint/convert-to-eslint-config.ts

```diff
@@ -0,0 +1,89 @@
+import type { Linter as ESLintLinter } from 'eslint';
+import {
+  createESLintConfiguration,
+  findReportedConfiguration,
+  joinConfigConversionResults,
+} from 'tslint-to-eslint-config';
+
+type TSLintRuleSeverity = 'warning' | 'error' | 'off';
+type TSLintRuleOptions = {
+  ruleArguments: any[];
+  ruleName: string;
+  ruleSeverity: TSLintRuleSeverity;
+};
```

Comment:
Oh we can export these types from `tslint-to-eslint-config` 😄. No need to force you to redeclare.

Reply:
It felt so minor I didn't bother to raise it

Reply:
For you @JamesHenry, anything 🎁 https://github.com/typescript-eslint/tslint-to-eslint-config/pull/816

Reply:
https://www.npmjs.com/package/tslint-to-eslint-config/v/2.0.0-beta3, released under the stable tag. 🚀